### PR TITLE
[plugin.video.southpark_unofficial] 0.4.4

### PR DESCRIPTION
**Pull request by wargio**
### Description
Fixed EN/DE/ES streams.
### Checklist:
- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] 1.0.0

### CODE REVIEW

File: plugin.video.southpark_unofficial/default.py [hunk 1]
```diff
@@ -287,10 +292,11 @@ def getCarousel():
 #	carousel = data_url.split("{resultsPerPage}/{currentPage}/json/{sort}")[0]
 #	carousel += "14/1/json/airdate"
 #	carousel += data_url.split("{resultsPerPage}/{currentPage}/json/{sort}")[1]
-	return "http://southpark.cc.com/feeds/carousel/video/351c1323-0b96-402d-a8b9-40d01b2e9bde/30/1/json/!airdate/promotion-0?lang="+audio
+	return "http://southpark.cc.com/feeds/carousel/video/2b6c5ab4-d717-4e84-9143-918793a3b636/14/2/json/!airdate/?lang="+audio
 
 def getMediagen(id):
 	feed = ""
+	print "http://"+mainweb_geo[audio_pos]+"/feeds/video-player/mrss/mgid:arc:episode:"+pageurl_geo[audio_pos]+":"+id+"?lang="+audio
```
Comment (MartijnKaijser) [inline plugin.video.southpark_unofficial/default.py]: leftover?

File: plugin.video.southpark_unofficial/default.py [hunk 2]
```diff
@@ -309,7 +315,7 @@ def getVideoData(mediagen):
 	if audio == "de":
 		mediagen += "&deviceOsVersion=4.4.4&acceptMethods=hls";
 		mediagen = mediagen.replace('{device}', 'Android')
-	print mediagen
+	print "MEDIAGEN: " + mediagen
```
Comment (MartijnKaijser) [inline plugin.video.southpark_unofficial/default.py]: please convert all these print statements to xbmc.log for next update.
https://codedocs.xyz/xbmc/xbmc/group__python__xbmc.html#ga3cd3eff2195727144ad6fbabd4744961

File: plugin.video.southpark_unofficial/changelog.txt [hunk 3]
```diff
@@ -46,4 +46,6 @@
 0.4.2
 - Fixed DE language (thanks to JinJin @kodi forums)
 0.4.3
-- Added Season 20
\ No newline at end of file
+- Added Season 20
+0.4.4
+- Fixed EN/DE/ES streams.
```
Comment (MartijnKaijser) [inline plugin.video.southpark_unofficial/changelog.txt]: It makes more sense to show newest changes at the top of the file as users see those first.

File: plugin.video.southpark_unofficial/default.py (hunk 1 above)
Comment (wargio) [inline plugin.video.southpark_unofficial/default.py]: ops. i'll remove it next release. sorry for that.

File: plugin.video.southpark_unofficial/default.py (hunk 2 above)
Comment (wargio) [inline plugin.video.southpark_unofficial/default.py]: I will do that with the next release! thanks! 👍

**Pull request merged**